compiler+runtime/test/cpp/jank/runtime/obj/persistent_string.cpp

```diff
@@ -0,0 +1,70 @@
+/* clang-format off */
+#include <jank/runtime/obj/persistent_string.hpp>
+#include <jank/runtime/obj/character.hpp>
+#include <jank/runtime/obj/number.hpp>
+#include <jank/runtime/obj/nil.hpp>
+#include <jank/runtime/core.hpp>
+
+/* This must go last; doctest and glog both define CHECK and family. */
+#include <doctest/doctest.h>
+
+namespace jank
```

Comment:
You're testing `jank::runtime::obj::persistent_string`, so we should be in `jank::runtime::obj`. That would allow you to get rid of all of the namespacing below, too.